### ISSUE TITLE
Editor in Desktop app not working - always redirects to webpage

### ISSUE THREAD

**Issue by drenjewski** [Automattic/wp-calypso#53220]
#### Steps to reproduce
1. open desktop app
2. connect to self hosted wordpress page
3. go to "Posts"
4. press button for new post

#### What I expected
Up to a certain version an editor opened directly in the app allowing me to write a new post, add media and publish

#### What happened instead
Whenever I press the "Write" button to create a new post or try to edit an existing post I get redirected to my self-hosted wordpress page in a browser instead of an editor being opened in the app.

#### App Version / OS version
6.8.0 / Windows 10 20H2

**Comment by anastas10s-afk** [comment]
The curious thing is that I get this same behavior in both the mobile and desktop app :/

mobile app version 17.1.0.3
iOS 14.5.1

**Comment by KirkwallDay** [comment]
Another instance of this issue here: 4015340-zen

It is a simple site, so no plugins, and they get the "You have one or more plugins that prevent editing content within Wordpress Deskstop continue in an external browser ?" error when trying to edit a page.

**Comment by devNigel** [comment]
Another report: 4021349-zen

**Comment by pauljacobson** [comment]
I have another report here: 4015991-zen

**Comment by Robertght** [comment]
Another report in #4030143-zen

**Comment by nsakaimbo** [comment]
Hello! Just a heads up that we're aware of this issue and are planning on releasing a complete overhaul of the application in a few weeks (currently in internal testing) that addresses this and myriad of other bugs. Once that version is released, we expect to be able to safely close this issue.

**Comment by xpurichan** [comment]
Received another report in 4037933-zd. This customer is using Linux Debain 10. They did mention that version 14 was working fine, and the problem began after updating to 15. Now versions 13, 14, and 15 do not work for them even with a clean install.

![Screen Shot 2021-06-02 at 10 52 44 AM](https://user-images.githubusercontent.com/33190975/120503003-e931e400-c390-11eb-915a-32cc58f4775d.jpg)

I have left their ticket on-hold so we can notify them once the fix is released.

**Comment by joashrajin** [comment]
Another report here: 4042200-zd-woothemes

They claim the issue only started happening recently.

**Comment by Robertght** [comment]
Another one in 4046543-zen

**Comment by swoodipto** [comment]
Just like @pauljacobson I have another user reporting that they see `Proceed in External Browser?" You have one or more plugins that prevent editing content within WordPress Desktop. Continue in an external browswer?` when trying to edit posts.

4045605-zen

As per a recent [Slack conversation](p1621332442002700/1619687868.010800-slack-C08GLCXMZ) the code has been merged but is under testing phase for now. But looks like the update was pushed but still this error persists?

**Comment by ersatzpenguin** [comment]
Another report of this issue in 4062763-zen.

**Comment by csonnek** [comment]
We have a number of users in this forums thread impacted by this as well: https://wordpress.com/forums/topic/getting-error-on-desktop-app/

**Comment by devNigel** [comment]
Another report: 28504214-hc

The user is using MacOS. They asked to be notified once the issue is fixed.

**Comment by JoshuaGoode** [comment]
Another report via 7372198-hc, on Windows.

**Comment by JoshuaGoode** [comment]
Related discussion: pbwngY-fZ-p2

TL;DR, Desktop app folks are aware and an updated version of the desktop app is still being worked on that will address this and a host of other issues.

**Comment by nsakaimbo** [comment]
To all on this thread: we're closing this issue as we expect this and a whole host of other issues to be resolved by the new version of the application (v7.0.5) that was just released. The latest build can now be downloaded at the WordPress Desktop landing page [here](P7jrAc-3-p2).

**Comment by kaitohm** [comment]
I can confirm, the original issue mentioned in this thread has now been resolved with the latest release - v7.0.5 . I'll go through and reply to all ticket instances noted on this thread, to notify users.

**Comment by csonnek** [comment]
@nsakaimbo I have someone [in this thread](https://wordpress.com/forums/topic/getting-error-on-desktop-app/page/2/#post-3701257) where 7.0.5 does not fix the issue - they end up getting a blank window in the app instead of their site dashboard. They updated the app and I had them delete and reinstall the app just to make sure with no effect.

OS: MacOS 10.15.7 Catalina
App version: 7.0.5

**Comment by nsakaimbo** [comment]
@csonnek Thanks for the report. However this is a different issue from the one described here initially, which we should track separately. 

Regardless, in the mean time it might help if we could get the user's application logs (Go to the "Help" menu and select "Get Application Logs"). This will output a zip archive that can be shared for inspection.

**Comment by KokkieH** [comment]
I've requested logs from the two people who have reported this new problem in 4126894-zen and 4126895-zen
